src/lighteval/tasks/templates/utils/translation_literals.py

```diff
@@ -349,4 +349,5 @@ def __getattribute__(self, name: str) -> str:
     Language.SORANI: TranslationLiterals(language=Language.SORANI),
```

Comment:
In the future, would be nice to sort this one alphabetically to avoid duplicates